companyLogin_Frontend

zanarkandlee opened the pull request:
meta data for companyLogin, added footer.php, edited style.css

zanarkandlee committed:
companyLogin_Frontend

meta data for companyLogin, added footer.php, edited style.css

wilsontanwm commented on traitquestdev/footer.php:
```diff
@@ -0,0 +1,28 @@
+<footer class="footer">
+	<div id="traitquestInfo" class="col-md-6">
```
i was thinking to add the details for index page, other pages will not have all the info as well as the contact us form to keep it simplified.

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -21,19 +32,24 @@
+				<input type="password" name="password" id="password" class="inputForm margin-align-center" placeholder="Password" />
+			</div>
+			<div id="loginResponse"></div>
+			<input type="submit" name="submit" id="loginSubmit" class="buttonForm button margin-align-center" value="Login" />
```
value for Submit button should be Login As Admin

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -21,19 +32,24 @@
+<div id="wrapper">
+	<?php include "header.php"; ?>
+	<div id="mainContainer" class="container">
+		<h1 class="text-center">Admin Login</h1>
```
no need title, the Submit button is clear enough to tell that it's login for admin

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -21,19 +32,24 @@
+		<h1 class="text-center">Admin Login</h1>
+		<form id="formCompanyLogin" class="form" method="post">
+			<div id="columnCompany" class="columnInput">
+				<input type="text" name="company" id="companyName" class="inputForm margin-align-center" placeholder="Company Name" />
```
change the placeholder to "Company"

wilsontanwm replied:
and no icon beside the placeholder? (applicable to all the columns)

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -12,7 +12,18 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```
missing meta data for title i think

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -12,7 +12,18 @@
+					CSS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+<link rel="stylesheet" type="text/css" href="css/bootstrap.css"/>
+<link rel="stylesheet" type="text/css" href="css/color.css"/>
```
please remove not-used color class in color.css

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -21,19 +32,24 @@
+				<input type="text" name="company" id="companyName" class="inputForm margin-align-center" placeholder="Company Name" />
+			</div>
+			<div id="columnEmail" class="columnInput">
+				<input type="text" name="email" id="email" class="inputForm margin-align-center" placeholder="Email" />
```
what is "margin-align-center" class doing? is it in any of the stylesheets?

wilsontanwm commented on traitquestdev/css/style.css:
```diff
@@ -26,11 +26,12 @@ select{}
-.inputForm{padding-top: 8px; padding-bottom: 8px;}
+.inputForm{width: 200px; padding:8px; display:block; margin: 0 auto;}
 .columnInput{margin-bottom: 5px;}
 .button{padding-top: 8px; padding-bottom: 8px; border: none;color:white; background-color: #2ECC71; cursor: pointer;transition: all 0.5s ease;-moz-transition: all 0.5s ease;-webkit-transition: all 0.5s ease;-o-transition: all 0.5s ease;-ms-transition: all 0.5s ease;}
```
please use the color in the sketch that i put in dropbox, i think the color code should be different.

wilsontanwm commented on traitquestdev/css/style.css:
```diff
@@ -51,8 +62,26 @@ textarea{resize:none;}
 ------------------------------------------ */
-#header{background:#C5EFF7;}
\ No newline at end of file
+#header{background:#C5EFF7;}
```
the header background color should be different compared to the one i designed in the dropbox?

zanarkandlee committed:
Updated #6_companyLogin_Frontend

edited according to comments made in github web
added fonts folder for glyphicons

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -35,21 +36,29 @@
 	<div id="mainContainer" class="container">
-		<h1 class="text-center">Admin Login</h1>
-		<form id="formCompanyLogin" class="form" method="post">
+		<h1 class="text-center" hidden>Admin Login</h1>
```
remove straight away, to keep the code clean

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -12,6 +12,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta name="title" content="Traitquest">
```
i believe it's TraitQuest with the capital Q, same applies to all other meta data

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -35,21 +36,29 @@
+			<input type="submit" name="submit" id="loginSubmit" class="buttonForm button" value="Login As Admin" />
+		</form>
 	</div>
-	<?php include "footer.php" ?>
```
what i mean here is maintain the footer.php, but the address, social network icons to be removed from footer.php

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -21,19 +33,32 @@
+			<div id="loginResponse"></div>
+			<input type="submit" name="submit" id="loginSubmit" class="buttonForm button" value="Login As Admin" />
+		</form>
+	</div>
```
to include footer.php but remove address, social network icons etc from footer.php

wilsontanwm commented on traitquestdev/companylogin.php:
```diff
@@ -21,19 +33,32 @@
+<div id="wrapper">
+	<?php include "header.php"; ?>
+	<div id="mainContainer" class="container">
+		<h1 class="text-center" hidden>Admin Login</h1>
```
to remove the h1 title straight away, dont have to hide it to keep the code clean

zanarkandlee committed:
Updated #6_companyLogin_Frontend v2

added footer.php to companyLogin.php
removed company details in footer.php
meta data for Traitquest has been changed to TraitQuest
removed h1 title on companyLogin.php
